internal/services/firewall/firewall_resource.go

```diff
@@ -224,6 +224,31 @@ func resourceFirewall() *pluginsdk.Resource {
 				},
 			},
 
+			"autoscale_configuration": {
+				Type:        pluginsdk.TypeSet,
+				Description: "Properties to provide a custom autoscale configuration to this azure firewall.",
```

Comment:
The description can be removed as this provider doesn't use it to generate the document.